modules/cpmso-ts-mhc-etcd-degraded.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="cpmso-ts-etcd-degraded_{context}"]
+= Recovering a degraded etcd Operator
+
+Certain situations can cause the etcd Operator to become degraded. 
+
+For example, while performing remediation, the machine health check might delete a control plane machine that is hosting etcd. If the etcd member is not reachable at that time, the etcd Operator becomes degraded. 
+
+When the etcd Operator is degraded, manual intervention is required to force the Operator to remove the failed member and restore the cluster state.
+
+.Procedure
+
+. List the control plane machines in your cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get machines -l machine.openshift.io/cluster-api-machine-role==master -n openshift-machine-api -o wide
+----
++
+Any of the following conditions might indicate a failed control plane machine:
++
+--
+** The `STATE` value is `stopped`.
+** The `PHASE` value is `Failed`.
+** The `PHASE` value is `Deleting` for more than five minutes.
```

Comment:
@jeana-redhat  just feel 5 minutes is a bit short. when I delete a master on aws cluster, it takes about 6 to 7 minutes from deleting to disappearing, it may take longer on other platforms. Not sure what the appropriate value is here.

Reply:
Ok - I am going to update this to say "ten minutes" and send it for editorial review, but I will wait to hear back from you if you think 10 is ok before merging. Thanks for the super quick reviews! :rocket: